tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -55,12 +55,6 @@
 
     well_known_endpoints.extend(pq_endpoints)
 
-
-# This can be removed when https://github.com/awslabs/s2n/issues/2220 is fixed.
-allowed_endpoints_failures = [
-    'wikipedia.org', 'yahoo.com'
```

Comment:
Don't we need to leave wikipedia? Or does it like Codebuild better than Travis?

Reply:
Wikipedia works now. I don't have a lot of context for why we turned it off in the first place, but I haven't seen it fail yet with this change.

Reply:
:/ I could, I don't love this idea because I don't think we should get into the habit of allowing failures in our tests again, given that it caused us to overlook this issue.